modules/cluster-logging-collector-log-forwarding-supported-plugins-5-2.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// logging/cluster-logging-external.adoc
+
+[id="cluster-logging-collector-log-forwarding-supported-plugins-5-2_{context}"]
+
+= Supported log data output types in OpenShift Logging 5.2
+
+Red Hat OpenShift Logging version 5.2 provides the following output types and protocols for sending log data to target log collectors.
+
+Red Hat tests each of the combinations shown in the following table. However, you should be able to send log data to a wider range target log collectors that ingest these protocols.
+
+[options="header"]
+|====
+| Output types   | Protocols          | Tested with
+
+| Amazon CloudWatch
```

Comment:
@syedriko @kabirbhartiRH Would you help me describe the protocol and version we test for this output?

Reply:
It's unclear which information from https://github.com/openshift/origin-aggregated-logging/blob/release-5.2/fluentd/Gemfile.lock belongs here.

Reply:
I don't think it makes a lot of sense to specify the protocol between fluentd CloudWatch plugin and the CloudWatch service. It's REST over HTTPS, but it doesn't really affect us or the customer. It's not like they have a choice in network settings of the CloudWatch backend. This is fundamentally different from, say, Kafka, where the customer deploys the Kafka broker.
What's important is that the versions of the AWS SDK that we ship in the currently supported versions of Logging can talk to the CloudWatch service running right now. And we're going to make sure that's the case. There's nothing for the customer to configure here.

Reply:
Updated. Please review and approve.

Reply:
/lgtm

Reply:
Approved by Sergey in Slack.